libcaja-private/caja-file.c

```diff
@@ -3495,6 +3504,11 @@ caja_file_compare_for_sort (CajaFile *file_1,
 				result = compare_by_full_path (file_1, file_2);
 			}
 			break;
+		case CAJA_FILE_SORT_BY_BTIME:
+			result = compare_by_time (file_1, file_2, CAJA_DATE_TYPE_CREATED);
+			if (result == 0) {
+				result = compare_by_full_path (file_1, file_2);
```

Comment:
No, idea if this is a valid warning, because you did the same like in other places of the code.
https://travis-ci.org/github/mate-desktop/caja/jobs/764737951#L4653
```
caja-file.c:3510:5: warning: Value stored to 'result' is never read [deadcode.DeadStores]

                                result = compare_by_full_path (file_1, file_2);

                                ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Maybe it can't be ignored?

Reply:
missing break statement...